src/mongocxx/collection.hpp

```diff
@@ -119,10 +118,8 @@ class MONGOCXX_API collection {
     ///
     /// @see http://docs.mongodb.org/manual/reference/command/aggregate/
     ///
-    cursor aggregate(
-        const pipeline& pipeline,
-        const options::aggregate& options = options::aggregate()
-    );
+    cursor aggregate(const pipeline& pipeline,
```

Comment:
There is a ticket slated for 0.4.0 to productionize clang-format, like we have done for the legacy driver and the server. In the meantime, we should do our best to keep the formatting in line with the current clang-format file in this branch manually.